content/en/admin/install.md

```diff
@@ -22,7 +22,7 @@ You will be running the commands as root. If you aren’t already root, switch t
 Make sure curl, wget, gnupg, apt-transport-https, lsb-release and ca-certificates are installed first:
```

Comment:
probably remove `apt-transport-https` here as you dropped it in the install line :-)